.github/workflows/release_docs_zip.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           ARCHIVE_FILE_NAME="broker-server-release-${GITHUB_REF#refs/tags/v}-deployment-docs.zip"
           echo "ARCHIVE_FILE_NAME=$ARCHIVE_FILE_NAME" >> $GITHUB_ENV
-          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env connector/.env connector/Dockerfile connector/README.md
+          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose-broker.yaml .env launchers/.env.broker launcher/Dockerfile connector/README.md
```

Comment:
let's merge the broker, 11000, 22000 should remain connectors, 33000 can become the broker. for this we should especially double check both the local development and local demo deployment guides and ensure that they contain all the ports, URLs, etc.

Reply:
Done